openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -150,12 +150,14 @@ class AccountViewSet(ViewSet):
 
     def list(self, request):
         """
-        GET /api/user/v1/accounts?usernames={username1,username2}
+        GET /api/user/v1/accounts?username={username1,username2}
         """
-        usernames = request.GET.get('usernames')
+        username = request.GET.get('username')
         try:
+            if username:
+                username = username[:-1].split(',') if username[-1:] == ',' else username.split(',')
```

Comment:
Simplify this with `username = username.strip(',').split(',')` or `username = [u for u in username.split(',') if u]`.